### PR TITLE
[DC-1792] Check if operational concepts are still present in the RDR export

### PR DESCRIPTION
**Pull request by rfrancis1**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/rdr_export_qc.py [hunk 1]
```diff
@@ -486,4 +486,52 @@
...
+'AZCArizonaSpecific_AZBannerIronwoodMedical','AZSArizonaSpecific_AZUofArizona','AZSArizonaSpecific_AZBannerGatewayMed',
+'AZSArizonaSpecific_AZBannerThunderbirdMed','AZSArizonaSpecific_AZBannerMDAnderson',
+'AZSArizonaSpecific_AZBannerDelEWebbMed','AZSArizonaSpecific_AZBannerBoswellMed','AZSArizonaSpecific_AZUAHN',
+'AZSArizonaSpecific_AZ_no','AZSArizonaSpecific_AZUPH','AZSArizonaSpecific_AZUofACancerCenter',
+'AZSArizonaSpecific_AZBannerDesertMed','AZSArizonaSpecific_AZ_yes','AZSArizonaSpecific_AZBannerUMed',
+'AZSArizonaSpecific_AZBannerEstrellaMed','AZSArizonaSpecific_AZUMI','AZCArizonaSpecific_AZBannerBaywoodMed',
+'AZCArizonaSpecific_AZBannerHeartHospital','AZCArizonaSpecific_CardonChildrensMedical','ExtraConsent_21YearsofAge',
+'21YearsofAge_21YearsYes','21YearsofAge_21YearsNo','ExtraConsent_19YearsofAge','19YearsofAge_19YearsYes',
+'19YearsofAge_19YearsNo','ReceiveCare_PIIState','VeteransHealthAdministration_Consent','VeteransHealthAdministration',
+'VeteransHealthAdministration_CareYes','VeteransHealthAdministration_CareNo','StateOfResidence','ExtraConsent_DNAVideo')
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: Is there a way to retrieve this full list (or most of it) from the Concept table with the right filters? I would prefer to reduce the number of hardcoded concepts

**Review by nishanthpp93 — COMMENTED** [review]
Thanks @rfrancis1 , this is helpful. I think it might be easier to maintain if the concepts are retrieved directly from the concept table. Let me know your thoughts.

File: data_steward/analytics/cdr_ops/rdr_export_qc.py (hunk 1 above)
Comment (rfrancis1) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: @nishanthpp93 so the list that the RDR team gets is from [this google document](https://docs.google.com/document/d/1q8XqUZJUWbEr4IXXdvzFmPV84qs8uLgbfZ6ffNGWWys/edit). Now, how do we best handle this list? say if someone decided to add more concepts to it?

Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: @rfrancis1 That's a good question. I'll raise this in eng sync and get back to you.

Comment (rfrancis1) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: @nishanthpp93 Did you have some feedback on this?

Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: @rfrancis1 There is a spreadsheet that we can use as a table that @lrwb-aou mentioned. However I think we can create a separate one for this purpose internal to the curation team. I'll set it up and reach out to you.

Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/rdr_export_qc.py:533]: @rfrancis1 I've set it up. Can you replace the subquery to use the table `operational_data.operational_ehr_consent`? I've also set up `operational_general_consent` if needed. They use a separate spreadsheet I've set up in the curation team drive.

**Review by nishanthpp93 — APPROVED** [review]
Looks great! Thanks for making the changes, Francis. There are some conflicts before merging so I will rebase and then merge.

**Pull request merged**